packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -108,7 +109,14 @@ class _LayoutDetailsTabState extends State<LayoutDetailsTab>
   Widget rootWidget(RemoteDiagnosticsNode node) {
     if (StoryOfYourFlexWidget.shouldDisplay(node))
       return StoryOfYourFlexWidget(controller);
-    return const SizedBox();
+    return const Center(
+      child: Text(
+        'Currently Layout Explorer only supports Flex-based widgets'
+        ' (e.g: Row, Column, Flex) or its direct children.',
```

Comment:
1. I believe the proper use of e.g. is  "e.g.,"
2. replace "its" with "their" since we are referencing a plural
3. then commenting again here in case my other commit got lost in the previous commit:
put comma after "Currently"